src/pages/paymentManagement/PaymentManagementPage.tsx

```
@@ -76,12 +78,15 @@ function PaymentManagementPage() {
     );
   }
 
-  const handleReject = (_payment: PaymentRequest) => {
-    // TODO: 입금 거절 API 연동
+  const { mutate: approvePayment } = useApprovePayment();
```

ESLint

[error] React Hook "useApprovePayment" is called conditionally. React Hooks must be called in the exact same order in every component render. Did you accidentally call a React Hook after an early return?

(react-hooks/rules-of-hooks)